capacity4more/modules/c4m/field/c4m_field_og/c4m_field_og.module

```diff
@@ -28,3 +28,172 @@ function c4m_field_og_ds_fields_info($entity_type) {
 
   return $fields;
 }
+
+/**
+ * Implements hook_field_access().
+ */
+function c4m_field_og_field_access($op, $field, $entity_type, $entity, $account) {
```

Comment:
@HelenaEksler Limit the items that are available depending on the current status and the possible statuses from there.

Not every user role can select any status option.

See https://wiki.amplexor.com/confluence/display/C4DNG/Group+approval+workflow